model/src/main/java/org/cbioportal/genome_nexus/model/StatsByTumorType.java

```diff
@@ -26,21 +26,24 @@
     private Double msiScore;
 
     @Field("n_with_sig")
-    private Integer nWithSig;
+    private Integer numberWithSig;
```

Comment:
The main reason I change this is somehow `nWithSig` will be changed to `nwithSig` in model, as well as `nGermlineHomozygous` (changed to `ngermlineHomozygous`), but other fields like `nCancerTypeCount` or `fBiallelic` are fine. I tried adding JsonProperty but didn't help, so I have to rename them.